### PR TITLE
Add description for compile-comments files

### PR DESCRIPTION
**Pull request by elken**
## Description
Adds a simple line for the compile-comments commands.

Refs #17

## Checklist
- [x] I have read
[CONTRIBUTING](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CONTRIBUTING.org)
guidelines.

#### [CHANGELOG](https://github.com/MarcoIeni/intellimacs/blob/master/docs/CHANGELOG.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [KEYBINDINGS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/KEYBINDINGS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

#### [PLUGINS](https://github.com/MarcoIeni/intellimacs/blob/master/docs/PLUGINS.org):
- [ ] Updated
- [ ] I will update it after this PR has been discussed
- [x] No need to update

### CODE REVIEW

File: spacemacs/compile-comments.vim [hunk 1]
```diff
@@ -1,27 +1,34 @@
...
 nnoremap <leader>cm    :action MakeModule<CR>
 vnoremap <leader>cm    :action MakeModule<CR>
 
 " Comment paragraph
+let g:WhichKeyDesc_Colors_CommentParagraphByLineComment = "<leader>cp     comment-paragraph-by-line-comment"
 nnoremap <leader>cp    vip:action CommentByLineComment<CR>
 vnoremap <leader>cp    <Esc>vip:action CommentByLineComment<CR>
 
 " Comment from begin of buffer up to current line
+let g:WhichKeyDesc_Colors_CommentBufferByLineComment = "<leader>ct     comment-buffer-start-to-line"
```
Comment (marcoieni) [inline spacemacs/compile-comments.vim]:
```suggestion
let g:WhichKeyDesc_Colors_CommentBufferByLineComment = "<leader>ct     comment-buffer-from-start-to-line"
```

Comment (marcoieni) [reply spacemacs/compile-comments.vim]: I don't know if you like it..

File: spacemacs/compile-comments.vim [hunk 2]
```diff
@@ -1,27 +1,34 @@
...
 nnoremap <leader>ct    Vgg:action CommentByLineComment<CR>
 vnoremap <leader>ct    <Esc>Vgg:action CommentByLineComment<CR>
 
 " Copy and comment current line
+let g:WhichKeyDesc_Colors_ = "<leader>cy     copy-current-line"
```
Comment (marcoieni) [inline spacemacs/compile-comments.vim]:
```suggestion
let g:WhichKeyDesc_Colors_CopyAndCommentCurrentLine = "<leader>cy     copy-and-comment-current-line"
```

**Comment by elken** [comment]
Done both and I just noticed I left in the "colors" :sweat_smile:

**Comment by marcoieni** [comment]
good catch 😂 Thank you very much :)

**Pull request merged**